examples/ui/ui-elements/list/keyboard-persist-tap/index.xml.njk

```diff
@@ -0,0 +1,48 @@
+---
+permalink: "/ui/ui-elements/list/keyboard-persist-tap/index.xml"
+tags: "List"
+hv_title: "Keyboard Persist Tap"
+hv_button_behavior: "back"
+---
+{% extends 'templates/base.xml.njk' %}
+
+{% block styles %}
+  <style
+    id="input"
+    borderBottomColor="#E1E1E1"
+    borderBottomWidth="1"
+    borderColor="#4E4D4D"
+    flex="1"
+    fontSize="16"
+    fontWeight="normal"
+    paddingBottom="8"
+    paddingTop="8">
+    <modifier pressed="true">
+    <style borderBottomColor="green"/>
+    </modifier>
+    <modifier focused="true">
+    <style borderBottomColor="#4778FF"/>
+    </modifier>
+  </style>
+{% endblock %}
+
+{% block container %}
+  <list keyboard-should-persist-taps="handled">
+    <item key="field" style="item">
+      <text-field name="field" style="input" auto-focus="true" placeholder="Tap!" />
```

Comment:
We now have the ability to create sudo component to abstract common XML markup, these live under `examples/_includes/` and can be included using the syntax `{% from ... import ... %}`.
Would you mind abstracting the text field and accompanying styles as a macro?

Reply:
Addressed in https://github.com/Instawork/hyperview/pull/940/commits/f366c4edd5b2da8cf7585c15b65cdb533070b3f6